Cargo.toml

```diff
@@ -15,6 +15,10 @@ exclude = [
     "tools",
 ]
 
+[workspace.dependencies]
+bolero = "0.12"
+bolero-generator = { version = "0.12", default-features = false }
```

Comment:
Cool this is a thing!

Wonder if we do this for other common deps also or what the criteria is for adding workspace deps. Test deps seem like good candidates.

Reply:
Yeah that's the plan. I just wanted to scope it down for now, though 😅